src/api/process.c

```diff
@@ -21,11 +22,16 @@
 #endif
 
 #define READ_BUF_SIZE 2048
+#define KILL_LIST_MIN_SIZE 16
```

Comment:
Is this used somewhere?

Reply:
Nope, old code that I'll remove once I got time.